src/symbols.c

```diff
@@ -1506,31 +1510,47 @@ static void show_goto_popup(GeanyDocument *doc, GPtrArray *tags, gboolean have_b
 	{
 		GtkWidget *item;
 		GtkWidget *label;
+		GtkWidget *box;
 		GtkWidget *image;
 		gchar *fname = short_names[i];
 		gchar *text;
-		gchar *sym = get_symbol_tooltip(doc, tmtag);
+		gchar *tooltip;
+		gchar *sym = get_symbol_tooltip(doc, tmtag, TRUE);
 
+		if (!sym)
+			sym = g_strdup(get_symbol_name(doc, tmtag, TRUE, FALSE));
 		if (!sym)
 			sym = g_strdup("");
+
 		if (! first && have_best)
-			/* For translators: it's the filename and line number of a symbol in the goto-symbol popup menu */
-			text = g_markup_printf_escaped(_("<b>%s:%lu:</b> %s"), fname, tmtag->line, sym);
+			/* For translators: it's the filename and the line number of a symbol in the goto-symbol popup menu */
+			text = g_markup_printf_escaped(_("<b>%s:%lu</b>"), fname, tmtag->line);
```

Comment:
Remove the translation mark so its not forgotten and spams next translate cycle, its computer language `file:line`, not human language, so its untranslatable.

Reply:
> Remove the translation mark so its not forgotten and spams next translate cycle, its computer language file:line, not human language, so its untranslatable.

@b4n Will French survive that? :-)

I agree with @elextr this is probably excessive so if it's OK with you, I'd remove those.

Reply:
Yeah as said in https://github.com/geany/geany/pull/3547#issuecomment-1752144085 now strings don't seem to hold much of anything that isn't idiomatic (`file:line` is a common scheme at least in programming, that don't need translating at least in French -- yet `file: line` was, as again, `file` was the label for `line`, notice the space which isn't idiomatic for the `file:line` notation)

Reply:
OK, done.